packages/angular/src/utils/lint.ts

```diff
@@ -0,0 +1,52 @@
+import { angularEslintVersion } from './versions';
+
+export const extraEslintDependencies = {
+  dependencies: {},
+  devDependencies: {
+    '@angular-eslint/eslint-plugin': angularEslintVersion,
+    '@angular-eslint/eslint-plugin-template': angularEslintVersion,
+    '@angular-eslint/template-parser': angularEslintVersion,
+  },
+};
+
+export const createAngularEslintJson = (
+  projectRoot: string,
+  prefix: string
+) => ({
+  overrides: [
+    {
+      files: ['*.ts'],
+      extends: ['plugin:@nrwl/nx/angular-code'],
+      parserOptions: {
+        project: [`${projectRoot}/tsconfig.*?.json`],
+      },
+      rules: {
+        '@angular-eslint/directive-selector': [
+          'error',
+          { type: 'attribute', prefix, style: 'camelCase' },
+        ],
+        '@angular-eslint/component-selector': [
+          'error',
+          { type: 'element', prefix, style: 'kebab-case' },
+        ],
+      },
+    },
+    {
+      files: ['*.html'],
+      extends: ['plugin:@nrwl/nx/angular-template'],
+      /**
+       * Having an empty rules object present makes it more obvious to the user where they would
+       * extend things from if they needed to
+       */
+      rules: {},
+    },
+    {
+      files: ['*.component.ts'],
+      extends: ['plugin:@angular-eslint/template/process-inline-templates'],
```

Comment:
To confirm, these rules _cannot_ be a part of the above `*.ts` files because it uses different rules (the ones that are for the templates above) that run on the inline templates?

I know you are aware of this but I'm not sure about limiting this to `*.component.ts` as people could write components in `component.spec.ts` and various other files as well. :thinking: Are there any options for configuring this so that these rules are a part of the first override?

Reply:
Per our discussion I have no fundamentally changed how angular-eslint works and this is no longer relevant